app.js

```diff
@@ -0,0 +1,8 @@
+function change( el )
+{
+    if ( el.value === "Sevak" )
```

Comment:
Use variables it will help you to avoid code reuse. 

E. g. `const firstUserName = "Sevak";`

`if ( el.value === firstUserName)`